authentication-service/src/main/kotlin/com/saveourtool/save/authservice/utils/SecurityUtils.kt

```diff
@@ -42,3 +64,26 @@ fun roleHierarchy(): RoleHierarchy = mapOf(
     .let {
         RoleHierarchyImpl().apply { setHierarchy(it) }
     }
+
+/**
+ * Get default [AuthorizationDecision] by [authentication] and [authorizationContext]
+ *
+ * @param authentication
+ * @param authorizationContext
+ * @return [Mono] of [AuthorizationDecision]
+ */
+fun defaultAuthorizationDecision(
```

Comment:
why do you extract this method?
maybe return back as a private function?

Reply:
private function is fine, will fix it

Reply:
Done